HPXMLtoOpenStudio/measure.rb

```diff
@@ -430,6 +442,21 @@ def self.set_defaults_and_globals(runner)
       water_heating_system.standby_loss = (2.9721 * sqft_by_gal - 0.4732).round(3) # linear equation assuming a constant u, F/hr
     end
 
+    # Default water heater location based on Building America climate zone
+    @hpxml.water_heating_systems.each do |water_heating_system|
+      if water_heating_system.location.nil?
+        location_hierarchy = get_location_hierarchy(@ba_cz_name)
+        water_heating_system.location = get_space_from_ba_location_hierarchy(location_hierarchy)
+      end
+    end
+
+    # Default water heater piping length
+    if @hpxml.water_heating_systems.size > 0
+      if @hpxml.hot_water_distributions[0].system_type == HPXML::DHWDistTypeStandard
+        @hpxml.hot_water_distributions[0].standard_piping_length = 50 # Chosen to test a negative EC_adj
```

Comment:
This needs to be inside an `if @hpxml.hot_water_distributions[0].standard_piping_length.nil?` statement.

Also, this isn't the right value. This value is simply an arbitrary value used in our sample test files. You will want to call the `HotWaterAndAppliances.get_default_std_pipe_length` method instead.